integration_tests/src/main/python/dpp_test.py

```diff
@@ -368,6 +368,7 @@ def setup_tables(spark):
                                ("spark.rapids.sql.hasExtendedYearValues", "false")]))
 
 
+@allow_non_gpu("EmptyRelationExec")  # 'EmptyRelationExec' is introduced from Spark 400
```

Comment:
Please create an issue to track and add the issue link here.

Reply:
Personally we have nothing to do with this new `EmptyRelationExec `.